apps/root/src/common/components/transaction-confirmation/index.tsx

```diff
@@ -217,7 +217,7 @@ const TransactionConfirmation = ({ shouldShow, handleClose, transaction, to, fro
         transactionReceipt?.type === TransactionTypes.swap
       ) {
         walletService
-          .getBalance(PROTOCOL_TOKEN_ADDRESS, transactionReceipt?.typeData.transferTo || transactionReceipt.from)
+          .getBalance(transactionReceipt?.typeData.transferTo || transactionReceipt.from, PROTOCOL_TOKEN_ADDRESS)
```

Comment:
did this actually change?

Reply:
There was an error that happened when updating balances after a swap. It wasn't related to this task, but while debugging my feature, I found out the problem was due to the parameters being in the wrong order.

Reply:
nice